printer_mods/README.md

```diff
@@ -110,6 +110,7 @@ like so:
 | | [Voron2.4_Pins_Mod](./hartk1213/Voron2.4_Pins_Mod) | This mod replaces all the M5 screws for all idlers/pulleys/bearings with 5mm pins  | :x:| :x: | :heavy_check_mark: |:x:|
 | | [Voron2.4_Y_Endstop_Relocation](./hartk1213/Voron2.4_Y_Endstop_Relocation) | This mod relocates the Y endstop to the top A drive piece, to work with Arkeets MGN12 mod  | :x:| :x: | :heavy_check_mark: |:x:|
 | henrikssn | [Plug Panel Mod for Generic Unfiltered Modules](./henrikssn/plug_panel_generic) | v2.4 plug panel for generic unfiltered modules with 66mm x 27mm cutout such as the Delta 06A2 and Qualtek 761-18 | :x: | :x: | :heavy_check_mark: | :x:|
+| henrikssn | [Euler Extruder for Voron AfterBurner](./henrikssn/euler_extruder) | The Euler extruder is a Sherpa Mini meets Clockwork, lightweight with full AfterBurner compatibility. | :x: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark:|
```

Comment:
Remove `henrikssn` from this line and just leave the cell blank, the name is already on the line above it.  See other people with multiple mods for examples.

Reply:
Done